### PR TITLE
Add jit stress modes to netci.groovy

### PR DESCRIPTION
**Pull request by sejongoh**
Added Jit stress options:
ForceRelocs
COMPlus_JITStress=2
COMPlus_JITStressRegs=1,2,3,4,8,0x10,0x80

### CODE REVIEW

**Comment by sejongoh** [comment]
@dotnet-bot test ci please

**Comment by sejongoh** [comment]
@mmitche Could you please review the changes?

File: netci.groovy [hunk 1]
```diff
@@ -28,7 +28,12 @@ class Constants {
     // This is a set of JIT stress modes combined with the set of variables that
     // need to be set to actually enable that stress mode.  The key of the map is the stress mode and
     // the values are the environment variables
-    def static jitStressModeScenarios = ['minopts' : ['COMPLUS_JitMinOpts' : '1'], 'jitstress1' : ['COMPLUS_JitStress' : '1'] ]
+    def static jitStressModeScenarios = ['minopts' : ['COMPLUS_JitMinOpts' : '1'], 'forcerelocs' : ['ForceRelocs' : '1'],
```
Comment (mmitche) [inline netci.groovy]: force relocs isn't COMPLUS?

**Comment by mmitche** [comment]
LGTM otherwise.

**Comment by mmitche** [comment]
@dotnet-bot test ci please (i think this might have been a spurious failure, or maybe two of these jobs running at the same time)

File: netci.groovy (hunk 1 above)
Comment (sejongoh) [inline netci.groovy]: You are right. COMPlus is needed. I thought that ForceReloc is an exception.

Comment (sejongoh) [inline netci.groovy]: Since Linux is case-sensitive, shouldn't the prefix be COMPlus?

Comment (janvorli) [inline netci.groovy]: Yes, it should be COMPlus

**Comment by mmitche** [comment]
@sejongoh LGTM once the other comments are fixed.

**Comment by ellismg** [comment]
FYI @sejongoh, your git email address is not configured to your Microsoft address, so the commits are not showed up as owned by your account.

**Comment by sejongoh** [comment]
@ellismg Thank you for finding it. I missed changing my git email address in my local machine.

**Pull request merged**